.travis.yml

```diff
@@ -28,6 +28,7 @@ install:
 
 script:
   - set -e
+  - regolith validate
```

Comment:
not sure how travis works but this looks ok.  Does this mean that it will run regolith validate?

Reply:
yes it should.